src/neo/SmartContract/Native/OracleContract.cs

```diff
@@ -179,7 +179,8 @@ internal override void PostPersist(ApplicationEngine engine)
             {
                 foreach (var (account, gas) in nodes)
                 {
-                    if (gas.Sign > 0) GAS.Mint(engine, account, gas, false);
+                    if (gas.Sign > 0)
+                        _ = GAS.Mint(engine, account, gas, false);
```

Comment:
this call should be awaited. Since it's void return, we need a try/catch block as well